### PR TITLE
fix: Add RagasLLM for customized test set generation

### PR DESCRIPTION
**Pull request by chankeith96**
Custom LLMs did not work for customized test set generation without first casting LLMs to RagasLLM

### CODE REVIEW

File: docs/concepts/testset_generation.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ from langchain.chat_models import ChatOpenAI
 # documents = load your documents
 
 # Add custom llms and embeddings
-generator_llm = ChatOpenAI(model="gpt-3.5-turbo")
-critic_llm = ChatOpenAI(model="gpt-4")
+generator_llm = LangchainLLM(llm=ChatOpenAI(model="gpt-3.5-turbo"))
```
Comment (jjmachan) [inline docs/concepts/testset_generation.md:69]: thank you so much for adding this but could you also import `LangchainLLM` from in line 61 with `from ragas.llms import LangchainLLM`

Comment (chankeith96) [reply docs/concepts/testset_generation.md:69]: Good catch, just added import statement

**Comment by jjmachan** [comment]
awesome, thank you so much for correcting the docs ❤️ 
and welcome to the community 🙂

**Comment by jjmachan** [comment]
Hey @chankeith96 
Thank you so much for helping us improve ragas with your PR ❤️ 
Now since it's Christmas and all we wanted to send you a postcard and a couple of stickers as our way of saying thank you for your contribution. If you are interested could you shoot an email at jamesjithin97@gmail.com and I'll tell you more?

Cheers 🙂 
Jithin

**Pull request merged**